news/_posts/2022-01-05-multistore-configuration-forms.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Multistore configuration forms"
+subtitle: "How multistore configuration forms got better with PrestaShop 1.7.8"
+date: 2022-01-05 
+authors: [matthieu-rolland]
+icon: icon-puzzle
+tags: [multistore, contribute]
+---
+
+With the release of PrestaShop 1.7.8, we expanded PrestaShop multistore capabilities to Symfony forms in order to build a reusable form component to make configuration forms multishop-compatible.
+
+Multishop affects a lof ot PrestaShop behaviors and components. One aspect of this feature is that user can use configuration forms to set different values for different shops, which is dealt with using shop contexts.
+
+## What does this mean
+
+In a configuration form, a merchant can set different values for different shop contexts, of which there are three, in this order of inheritance:
+
+- All shop context
+- Group shop context
+- Single shop context
+
+![Shop contexts](/assets/images/2022/01/shopcontexts.png)
+
+When a value is not set for a context, it inherits the value of the parent context. Consequently, the "All Shops" context values can be considered a default values.
+
+We listened to merchants feedback and were told this mechanism was not obvious, and it was not easy to see at a glance which form fields were modified or not for the current context.
```

Comment:
We must try to avoid "we" when possible because it's confusing

```suggestion
Product designers listened to merchants feedback and were told this mechanism was not obvious, and it was not easy to see at a glance which form fields were modified or not for the current context in the back-office.
```